test/rendering-test/test-rendering.js

```diff
@@ -113,7 +113,7 @@ class RenderingTest extends Component {
     });
   }
 
-  _onDrawComplete(name, referecenResult, canvas) {
+  _onDrawComplete(name, referecenResult, callbackData) {
```

Comment:
Nit: We usually call these something like simple like `opts`. `callbackData` is more C style. But maybe a good idea...

Nit: `referecenResult` is misspelled

Reply:
thanks.